### ISSUE TITLE
Need help! learning_rate

### ISSUE THREAD

**Issue by soyusora** [Linaqruf/kohya-trainer#52]
No matter how I try to change the values, my learning_rate does not change and remains like "2e-06" in metadata. And I need 1e-4, as well as unet lr (btw text_encoder_lr:  "5e-05"). I don't understand at all what I'm doing wrong. I use kohya-LoRA-finetuner.ipynb

**Comment by Linaqruf** [comment]
dont need to worry about learning rate printed in metadata if you already set unet_lr and text_encoder_lr

**Comment by Linaqruf** [comment]
i put the learning rate back to the training cell, but still in dev branch

**Comment by soyusora** [comment]
> i put the learning rate back to the training cell, but still in dev branch

ok, thank you)
